_site/index.html

```diff
@@ -588,18 +599,19 @@ <h3>T-Z</h3>
           </div>
         </section>
 
-        <section>
+        <section class="block__word list">
+          <p class="word__signal"></p>
           <p class="word__title" id="non-binary">
-            Non-Binary
+            Non-binary
             <span class="word__speech">
               adj
             </span>
           </p>
           
           <p class="word__definition">
-            someone whose gender does not fit into the male-female gender binary.<br><br>
+            an umbrella term covering any gender does not fit into the male-female gender binary.<br><br>
```

Comment:
of, relating to, or characterized by being any gender does not fit into the male-female gender binary; an umbrella term.